### ISSUE TITLE
Development files included in Composer installation

### ISSUE THREAD

**Issue by edmundcwm** [PatelUtkarsh/better-file-name-ai#14]
## Description
When installing the plugin via Composer the development files are also included:

![Screenshot 2023-12-05 at 12 41 48 PM](https://github.com/PatelUtkarsh/better-file-name-ai/assets/13222067/d5617d19-061d-4fbb-a353-0d451b559dea)

Should we add a `.gitattributes` and `export-ignore` those files?

**Comment by PatelUtkarsh** [comment]
Good idea @edmundcwm, 

Maybe we can use archive exclude https://getcomposer.org/doc/04-schema.md#archive ?

**Comment by PatelUtkarsh** [comment]
Composer https://getcomposer.org/doc/04-schema.md#archive did not work, Yet to deep dive into why but likely because zip is actually created by GitHub and not composer. 

Fixed in #18 using `.gitattributes` which is working as expected. Thanks @edmundcwm. 

```
.
├── better-file-name.php
├── composer.json
├── composer.lock
├── readme.md
├── readme.txt
└── src
    ├── Admin.php
    ├── Generate_Alt_Text_Cli.php
    ├── Image.php
    ├── Openai_Wrapper.php
    └── Settings.php

2 directories, 10 files
```
